### PR TITLE
Modules Page 

### PR DESCRIPTION
**Pull request by AreYouIvan**
Hello this PR solves issue #6

Create the page where all the modules will be shown. Every module redirects to the page to start working.
Also, all the cards should have a description a default image, and a title.

Acceptance Criteria

 Create the Page "Modules" or "Admin"
 Create the component "module card"
 All the cards must redirect to the corresponding page
 The page should have styles applied
(There's a bug with the icons that hadn't been solved :( )
image

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -3,10 +3,19 @@ module.exports = {
...
     "./src/components/*/*.{js,ts,jsx,tsx}",
     "./src/services/*/*.{js,ts,jsx,tsx}",
-    "./src/utils/*/*.{js,ts,jsx,tsx}"
+    "./src/utils/*/*.{js,ts,jsx,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      backgroundImage: {
+        "users-icon": "url('../assets/icons/users-icon.svg')",
```
Comment (Hai5edfm) [inline tailwind.config.js:11]: if we want to add more icons or images... will we have to add them all here?

Comment (AreYouIvan) [reply tailwind.config.js:11]: Only if you will call them as background images

**Review by Hai5edfm — APPROVED** [review]

File: src/pages/modules/index.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      </Head>
+      {/* <Header></Header> */}
+      <h1 className="p-2 mx-8 mt-5 text-3xl font-semibold text-gray-900 border-l-4 border-blue-500 sm:text-left sm:ml-32">
+        {TITLE}
+      </h1>
+      <section className="flex flex-col items-center justify-center my-8">
+        <ModuleCard
+          title={"Users"}
+          description={"Find , create, edit or delete users"}
+          icons={"users-icon"}
```
Comment (LilianaRestrepoTorres) [inline src/pages/modules/index.tsx:20]: What is 'users-icon' here? It probably is the problem.

**Review by LilianaRestrepoTorres — APPROVED** [review]
It looks great!
Nicely done. :snowboarder:

**Pull request merged**